File: Enums/Enums.cs
namespace Uno.Enums
{
    public enum CardColor
    {
        Blue = 1,
        Green = 2,
        Red = 3,
        Yellow = 4,
        Wild = 5
    }

    public enum CardValue
    {
        Zero = 0,
        One = 1,
        Two = 2,
        Three = 3,
        Four = 4,
        Five = 5,
        Six = 6,
        Seven = 7,
        Eight = 8,
        Nine = 9,
        Reverse = 10,
        Skip = 11,
        DrawTwo = 12,
        DrawFour = 13,
        ChangeColor = 14,
        BlackHole = 15,
        DiscardWildCards = 16,
        StealTurn = 17,
        SwapHands = 18,
        DoubleEdge = 19,
        DiscardColor = 20,
        HandOfGod = 21,
        Judgement = 22,
        UnitedWeFall = 23,
        ParadigmShift = 24,
        Deflect = 25,
        InspectHand = 26,
        GraveDigger = 27,
        RussianRoulette = 28,
        Roulette = 29,
        Duel = 30,
        KeepMyHand= 31

    }

    public enum Direction
    {
        Right = 1,
        Left = 2
    }

    public enum TypeOfMessage
    {
        Chat = 1,
        Server = 2,
        Spectators = 3
    }
    public enum GameMode
    {
        Normal = 1,
        SpecialCards = 2,
        SpecialCardsAndAvalonCards = 3
    }

    public enum ChatMessageIntention
    {
        Normal = 1,
        Buzz = 2
    }

}

File: ClientApp/src/app/_models/enums.ts
export enum TypeOfMessage {
  chat = 1,
  server = 2,
  spectators = 3
}

export enum CardColor {
  blue = 1,
  green = 2,
  red = 3,
  yellow = 4,
  wild = 5
}

export enum CardValue {
  zero = 0,
  one = 1,
  two = 2,
  three = 3,
  four = 4,
  five = 5,
  six = 6,
  seven = 7,
  eight = 8,
  nine = 9,
  reverse = 10,
  skip = 11,
  drawTwo = 12,
  drawFour = 13,
  changeColor = 14,
  blackHole = 15,
  discardAllWildCards = 16,
  stealTurn = 17,
  swapHands = 18

}

export enum Direction {
  right = 1,
  left = 2
}

export enum GameMode {
  normal = 1,
  specialCards = 2,
  specialCardsAndAvalonCards = 3
}
